### PR TITLE
Update to KF6

### PR DESCRIPTION
**Pull request by sungsphinx**
Needs testing, I just need to sort out my `flatpak-builder`.

EDIT - 10th March 5:51: WebView doesn't work right now, but I have an idea.

### CODE REVIEW

File: org.kde.tokodon.json [hunk 1]
```diff
@@ -13,7 +15,9 @@
         "--share=network",
         "--talk-name=org.freedesktop.Notifications",
         "--talk-name=org.kde.kwalletd5",
-        "--talk-name=org.freedesktop.secrets"
+        "--talk-name=org.kde.kwalletd6",
```
[the remark this answers is not in the document]
Comment (travier) [reply org.kde.tokodon.json:18]: my bad, that's what you did

**Comment by bbhtt** [comment]
> EDIT - 10th March 5:51: WebView doesn't work right now, but I have an idea.

`flatpak run --env=QT_PLUGIN_PATH="/app/plugins:/app/lib64/plugins:/app/lib/plugins:/usr/share/runtime/lib/plugins" --env=LD_LIBRARY_PATH=/app/lib/x86_64-linux-gnu org.kde.tokodon` if you are on aarch64 change it accordingly

I'll figure out how to fix this in runtime and baseapp.

**Comment by bbhtt** [comment]
bot, build

**Comment by bbhtt** [comment]
Added a fix to the [baseapp](https://github.com/flathub/io.qt.qtwebengine.BaseApp/commit/c399e93ae606cf6d837d3ac2fec42153c31c5a01), just needs the [runtime bit](https://invent.kde.org/packaging/flatpak-kde-runtime/-/merge_requests/201) now

**Comment by travier** [comment]
```
No WebView plug-in found!
```

**Comment by travier** [comment]
```
$ flatpak run --env=QT_PLUGIN_PATH="/app/plugins:/app/lib64/plugins:/app/lib/plugins:/usr/share/runtime/lib/plugins" org.kde.tokodon
```

Works for me so let's add that temporarily as environment variable.

**Comment by travier** [comment]
This workaround works for me so I'd say we do this until we figure out something better.

**Review by travier — APPROVED** [review]

**Pull request merged**